pom.xml

```diff
@@ -29,8 +29,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <!-- DependencyGraph is final in Jenkins < 1.425 -->
-        <version>1.425</version>
+        <version>1.532.3</version>

```

Comment:
No reason to require such ancient core, this enforces java 1.5 code and doesn't allow to use 1.6 features. 1.532.3 is the first LTS (after 1.509) that allows to use 1.6, i can revert to 1.532, but in production there is no sense to use 1.532.1 when there is 1.532.3 with many fixes.